woocommerce/payment-gateway/External_Checkout/Admin.php

```diff
@@ -288,6 +291,192 @@ protected function get_display_location_options() {
 	}
 
 
+	/**
+	 * Adds admin notices for configuration issues.
+	 *
+	 * @since 5.10.0
+	 */
+	public function add_admin_notices() {
+
+		// if the feature is not enabled, bail
+		if ( ! $this->handler->is_enabled() ) {
+			return;
+		}
+
+		// if not on the settings screen, bail
+		if ( ! $this->is_settings_screen() ) {
+			return;
+		}
+
+		$this->add_configuration_errors_notice();
+
+		$this->add_configuration_warnings_notices();
+	}
+
+
+	/**
+	 * Adds one error notice for all configuration options that need attention.
+	 *
+	 * @since 5.10.0
+	 */
+	protected function add_configuration_errors_notice() {
+
+		$errors = $this->get_configuration_errors();
+
+		if ( ! empty( $errors ) ) {
+
+			$message = sprintf(
+				/* translators: Placeholders:  - external checkout label, %2$s - <strong> tag, %3$s - </strong> tag */
+				__( '%2$S%1$s is disabled.%3$S', 'woocommerce-plugin-framework' ),
+				$this->handler->get_label(),
+				'<strong>', '</strong>',
```

Comment:
This trailing comma causes a fatal in PHP 7.2
```suggestion
				'<strong>', '</strong>'
```